app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -414,20 +422,40 @@ export default {
         :childClans="currentSelectedClanChildDetails"
         class="clan-search"
       />
-      <InputClanSearch v-if="isGlobalPage" max-width="510" style="margin: 10px auto"/>
+      <InputClanSearch v-if="isGlobalPage" :max-width="510" style="margin: 10px auto"/>
       <p class="subheader2">{{ $t('league.ladder_subheader') }}</p>
       <div class="col-lg-6 section-space">
-        <leaderboard v-if="currentSelectedClan" :rankings="selectedClanRankings" :playerCount="selectedClanLeaderboardPlayerCount" :key="`${clanIdSelected}-score`" :clanId="clanIdSelected" class="leaderboard-component" style="color: black;" />
-        <leaderboard v-else :rankings="globalRankings" :playerCount="globalLeaderboardPlayerCount" class="leaderboard-component" />
+
+        <Leaderboard
```

Comment:
Ended up making these changes as I was passing in the students via props first. Can drop this if it is not useful, but it helps me see the exact difference between the two conditional leaderboard components. 

Also updated to capitalize, to follow the rest of the file.